harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -582,11 +582,21 @@ def _load(self) -> None:
             ["checkpoint.pt"],
         ]
 
+        checkpoint: Optional[Dict[str, Any]] = None
         for ckpt_path in potential_paths:
             maybe_ckpt = self.load_path.joinpath(*ckpt_path)
             if maybe_ckpt.exists():
                 checkpoint = torch.load(str(maybe_ckpt), map_location="cpu")  # type: ignore
                 break
+        if checkpoint is None or not isinstance(checkpoint, dict):
+            # CHECK me
```

Comment:
it seems like all checkpoints would be an instance of dict, correct?

Reply:
yes.